pyvista/core/grid.py

```diff
@@ -286,26 +312,22 @@ def z(self):
     def z(self, coords):
         """Set the coordinates along the Z-direction."""
         self.SetZCoordinates(numpy_to_vtk(coords))
+        self._update_dimensions()
         self.Modified()
 
 
-    # @property
-    # def quality(self):
-    #     """
-    #     Computes the minimum scaled jacobian of each cell.  Cells that have
-    #     values below 0 are invalid for a finite element analysis.
-    #
-    #     Returns
-    #     -------
-    #     cellquality : np.ndarray
-    #         Minimum scaled jacobian of each cell.  Ranges from -1 to 1.
-    #
-    #     Notes
-    #     -----
-    #     Requires pyansys to be installed.
-    #
-    #     """
-    #     return UnstructuredGrid(self).quality
+    @Grid.dimensions.setter
```

Comment:
If I understand correctly this overrides the setter on `Grid.dimensions` (for `Grid` and any subtype), so I'd change the that directly instead.

Reply:
Other classes use the setter for `Grid.dimensions`, only `RectilinearGrid`s have this blocked

Reply:
it will not override other subtypes of `Grid`.... i think...

Reply:
`test_uniform_setters` is passing so this should be fine

Reply:
Ah cool, learned something - I didn't know overriding just setters of parent properties worked. Thanks!

Reply:
Awesome, thanks for checking! This was the first time I had overridden a property setter like that too